### DISCUSSION TITLE
使用多个workers

### DISCUSSION THREAD

**Discussion by ffleonardo** [SeaHOH/GotoX#220] [问答/咨询] [answered]
如题，在配置文件中有如下：
#如果你部署了多个 worker，那么请将 .workers.dev 下的子域名 (仅一个) 和 worker 名称填入
#这两个设置将覆盖上面的 worker 设置，无法使用不同账户下的 worker 和绑定的域名
#每个 worker 提供 128MB 运存，6 个并发处理，请按需部署添加
#多个 worker 并不会增加免费配额，仅为增加请求队列使代理更流畅
subdomain =
workers = 
请问要如何填写多个workers至"workers"下？

**Comment by SeaHOH** [comment, marked as answer]
假如你建立了如下几个 Workers：
```
worker-name1.your-subdomain.workers.dev
worker-name2.your-subdomain.workers.dev
worker-name3.your-subdomain.workers.dev
```
配置如下：
```ini
subdomain = your-subdomain
workers = worker-name1|worker-name2|worker-name3
```
